docs/user-guide/environment-variables.md

```diff
@@ -44,6 +44,7 @@ _Note: Environment variables set in one job cannot be accessed in another job. T
 |------|-------|
 | SD_SOURCE_DIR | Location of checked-out code |
 | SD_ARTIFACTS_DIR | Location of built/generated files |
+| SD_SOURCE_PATH | Location of matched source path. See [Source Paths](./configuration/sourcePaths). |
```

Comment:
~Location of matched source path~`The source path(s) that triggered this build to run`?

Reply:
@tkyi it will be only one value, so `path(s)` is not correct. I will update the rest.

Reply:
Why will it only be one value? Users can specify multiple source paths in their yaml, and it's possible for multiple source paths to match, right?